README.riscos

```diff
@@ -11,19 +11,17 @@ perlriscos - Perl version 5 for RISC OS
 =head1 DESCRIPTION
 
 This document gives instructions for building Perl for RISC OS. It is
-complicated by the need to cross compile. There is a binary version of
-perl available from L<http://www.cp15.org/perl/> which you may wish to
+complicated by the need to cross-compile. There is a binary version of
+perl available from L<https://www.cp15.org/perl/> which you may wish to
 use instead of trying to compile it yourself.
 
 =head1 BUILD
 
-You need an installed and working gccsdk cross compiler
-L<http://gccsdk.riscos.info/> and REXEN
-L<http://www.cp15.org/programming/>
+You need an installed and working L<GCCSDK|https://gccsdk.riscos.info/>
```

Comment:
The `https://gccsdk.riscos.info/` URL redirects to a 404.

I suspect http://www.riscos.info/downloads/gccsdk/ is the best current option.

Reply:
Yes, will amend to https://www.riscos.info/downloads/gccsdk/latest/ then